google/cloud/storage/internal/grpc/default_options.cc

```diff
@@ -31,6 +31,8 @@ namespace storage_internal {
 GOOGLE_CLOUD_CPP_INLINE_NAMESPACE_BEGIN
 namespace {
 
+auto constexpr kMinMetricsPeriod = std::chrono::seconds(60);
```

Comment:
(just curious) Where did this value come from? The GCM minimum is a 5 second period.

Reply:
I made it up, fixed.